webplugin/js/app/kommunicate.js

```
@@ -196,9 +196,13 @@ $applozic.extend(true, Kommunicate, {
         KommunicateUI.showChat();
         KommunicateUI.hideFaq();
     },
-    openConversation: function (groupId) {
+    openConversation: function (groupId, groupDetail) {
         kommunicateCommons.setWidgetStateOpen(true);
-        window.$applozic.fn.applozic('loadGroupTab', groupId);
+        if (groupId) {
+            window.$applozic.fn.applozic('loadGroupTab', groupId);
+        } else if (groupDetail && groupDetail.clientGroupId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)